process/transactionLog/process.go

```diff
@@ -53,7 +55,8 @@ func NewTxLogProcessor(args ArgTxLogProcessor) (*txLogProcessor, error) {
 	return &txLogProcessor{
 		storer:      storer,
 		marshalizer: args.Marshalizer,
-		logs:        make(map[string]data.LogHandler),
+		logs:        make([]indexer.LogData, 0),
+		logsIndices: make(map[string]int),
```

Comment:
Strongly suggest to use slices of pointers for the logs,  `map[string]*indexer.LogData` for the logsMap on L59
reasons: 
1. `for...range` on a slice of structs does not behave as expected, can end up with hard to catch bugs
2. we are avoiding copying the same data over and over again. Using pointers we will copy only a uint64 value.

Reply:
refactored